endpoints/getting-started/clients/google-jwt-client.py

```diff
@@ -68,9 +68,8 @@ def make_jwt_request(signed_jwt, url='https://your-endpoint.com'):
         'content-type': 'application/json'
     }
     response = requests.get(url, headers=headers)
+    print(response.status_code, response.content)
```

Comment:
```suggestion
    print(response.status_code, response.content)
    response.raise_for_status()
```

Hi - could you re-add `raise_for_status()` so an exception is thrown if the request fails?

Reply:
Done